src/components/App.js

```diff
@@ -0,0 +1,10 @@
+import { Provider } from 'react-redux';
+import store from '../Redux/store/index';
+
+
+const App = ({ store }) => (
+  <Provider store={store}>
+
+  </Provider>
+)
+ReactDOM.render(<App store={store} />, document.getElementById('app'));
```

Comment:
I recommend you to move `App.js` from component to root of `src/`directory. `App.js` should be independent from both components and redux directories.

For more info, please observe the folder structure.
<img width="654" alt="Screen Shot 2019-12-23 at 19 42 12" src="https://user-images.githubusercontent.com/33821734/71372272-8c87a280-25bc-11ea-9541-3ab8961dd866.png">